### DISCUSSION TITLE
Multiple Marker Shapes in one Geomap

### DISCUSSION THREAD

**Discussion by asayeedi** [grafana/grafana#38740] [Dashboards, Panels & Visualizations]
I'd love the possibility to have the multiple markers like Circle and Square in same Geomap.
Suppose I have a Device Type A and Type B and I want to show Device Type A in Geomap with Circle and Device Type B with Square. It becomes very convenient to Identify the types without hovering over each markers.

**Comment by nmarrs** [comment]
Hi @asayeedi, Geomap now supports adding multiple layers which supports having multiple maker layers. See latest [geomap docs](https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/geomap/) for more info

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
